Tasks/Common/ios-signing-common/ios-signing-common.ts

```diff
@@ -225,20 +224,31 @@ export async function getProvisioningProfileUUID(provProfilePath: string) {
     })
     await plistTool.exec();
 
+    //use PlistBuddy to figure out the Name
+    let provProfileName: string;
+    plistTool = tl.tool(plist);
+    plistTool.arg(['-c', 'Print Name', tmpPlist]);
+    plistTool.on('stdout', function (data) {
+        if (data) {
+            provProfileName = data.toString();
+        }
+    })
+    await plistTool.exec();
+
     //delete the temporary plist file
     let deletePlistCommand: ToolRunner = tl.tool(tl.which('rm', true));
     deletePlistCommand.arg(['-f', tmpPlist]);
     await deletePlistCommand.exec();
 
-    if (provProfileUUID) {
+    if (provProfileUUID && provProfileName) {
```

Comment:
Can this break existing builds? Can there be profiles that don't have a name? I don't think so but good to double check.